docs/1.-How-to-Use-JPlag.md

```diff
@@ -106,6 +106,19 @@ Subsequence Match Merging
       --required-merges=<minimumRequiredMerges>
                         Minimal required merges for the merging to be applied
                           (between 1 and 50, default: 6).
+
+Frequency Analysis
+      --isFrequencyAnalysisEnabled-analysis
```

Comment:
see coment above. Name should probably be more like --is-frequency-analysis-enabled